fbw-common/src/wasm/systems/systems/src/air_conditioning/acs_controller.rs

```diff
@@ -2159,18 +2151,30 @@ mod acs_controller_tests {
                         context,
                         AcscId::Acsc1(Channel::ChannelOne),
                         &cabin_zones,
-                        vec![
-                            ElectricalBusType::DirectCurrent(1),
-                            ElectricalBusType::AlternatingCurrent(1),
+                        [
+                            [
+                                ElectricalBusType::AlternatingCurrent(1), // 103XP
+                                ElectricalBusType::DirectCurrent(1),      // 101PP
+                            ],
+                            [
+                                ElectricalBusType::AlternatingCurrent(2),  // 202XP
+                                ElectricalBusType::DirectCurrentEssential, // 4PP
+                            ],
                         ],
                     ),
                     AirConditioningSystemController::new(
                         context,
                         AcscId::Acsc2(Channel::ChannelOne),
                         &cabin_zones,
-                        vec![
-                            ElectricalBusType::DirectCurrent(2),
-                            ElectricalBusType::AlternatingCurrent(2),
+                        [
+                            [
+                                ElectricalBusType::AlternatingCurrent(2), // 101XP
+                                ElectricalBusType::DirectCurrent(2),      // 103PP
+                            ],
+                            [
+                                ElectricalBusType::AlternatingCurrent(2), // 204XP
+                                ElectricalBusType::DirectCurrent(2),      // 206PP
```

Comment:
I think this is a copy paste error?

Reply:
Fixed here too as per other comment. Nice find, thanks.